docs/sources/tasks/collect-logs-in-kubernetes.md

```diff
@@ -0,0 +1,323 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/tasks/collect-logs-in-kubernetes/
+description: Learn how to collect and forward logs from Kubernetes to Loki
+title: Collect and forward Kubernetes logs
+weight: 200
+---
+
+# Collect logs and forward Kubernetes logs
+
+You can configure {{< param "PRODUCT_NAME" >}} to collect logs and forward them to a [Loki][] database.
+
+This topic describes how to:
+
+* Configure logs delivery.
+* Collect logs from Kubernetes Pods.
+
+## Components used in this topic
+
+* [discovery.kubernetes]
+* [discovery.relabel]
+* [local.file_match]
+* [loki.source.file]
+* [loki.source.kubernetes]
+* [loki.source.kubernetes_events]
+* [loki.process]
+* [loki.write]
+
+## Before you begin
+
+* Ensure that you have basic familiarity with logs labelling when working with Loki.
+* Identify where you will write collected logs.
+  Logs can be written to Loki endpoints such as Grafana Loki, Grafana Cloud, or Grafana Enterprise Logs.
+* Be familiar with the concept of [Components][] in {{< param "PRODUCT_NAME" >}}.
+
+## Configure logs delivery
+
+Before components can collect logs, you must have a component responsible for writing those logs somewhere.
+
+The [loki.write][] component is responsible for delivering logs to a Loki endpoint.
+After a `loki.write` component is defined, you can use other {{< param "PRODUCT_NAME" >}} components to forward logs to it.
+
+To configure a `loki.write` component for logs delivery, complete the following steps:
+
+1. Add the following `loki.write` component to your configuration file.
+
+   ```alloy
+   loki.write "<LABEL>" {
+     endpoint {
+       url = "<LOKI_URL>"
+     }
+   }
+   ```
+
+   Replace the following:
+
+   - _`<LABEL>`_: The label for the component, such as `default`.
+     The label you use must be unique across all `loki.write` components in the same configuration file.
+   - _`<LOKI_URL>`_ The full URL of the Loki endpoint where logs will be sent, such as `https://logs-us-central1.grafana.net/loki/api/v1/push`.
+
+1. If your endpoint requires basic authentication, paste the following inside the `endpoint` block.
+
+   ```alloy
+   basic_auth {
+     username = "<USERNAME>"
+     password = "<PASSWORD>"
+   }
+   ```
+
+   Replace the following:
+
+   - _`<USERNAME>`_: The basic authentication username.
+   - _`<PASSWORD>`_: The basic authentication password or API key.
+
+1. If you have more than one endpoint to write logs to, repeat the `endpoint` block for additional endpoints.
+
+The following example demonstrates configuring `loki.write` with multiple endpoints and mixed usage of basic authentication, and a `loki.source.file` component which forwards logs to it.
```

Comment:
```suggestion
The following example demonstrates configuring `loki.write` with multiple endpoints, mixed usage of basic authentication, and a `loki.source.file` component that forwards logs to it.
```

What is the `it` here? Is it `loki.write`?

Reply:
I believe it's Loki (as the database). It's a copy paste from Prometheus task.

Reply:
OK, then we could replace `it` with `Loki`.